backend/app/api/v1/commons/hasher.py

```diff
@@ -0,0 +1,41 @@
+import zlib
+import hashlib
+from cryptography.fernet import Fernet
+
+symmetric_encryptor = b'k3tGwuK6O59c0SEMmnIeJUEpTN5kuxibPy8Q8VfYC6A='
```

Comment:
not sure what this is used for, but I am curious should it be out in the open?

Reply:
Telco data doesn't have an UUID attribute. So I am generating a digest by encypting and compressing it using this constant key. The idea is to feed that encrypted data as an input to `/graph` endpoint to display a ploty visual if the run has met/not met a particular threshold. This key is just transforming the fetched data into an intermediate format so that we can feed it as an input to another function, so I think it should be okay. Open to other thoughts too.